templates/items/consumable.hbs

```diff
@@ -110,7 +110,9 @@
                 {{/if}}
                 {{#each properties}}
                 <label class="checkbox">
-                    <input type="checkbox" name="system.properties.{{@key}}" {{checked selected}}> {{ label }}
+                    <input type="checkbox" name="system.properties.{{ @key }}" {{ checked selected }} {{#if disabled~}}
```

Comment:
Does the code in `ItemSheet5e#activateListeners` not do this automatically?

Reply:
Indeed